cmd/copyondemand/main.go

```diff
@@ -78,9 +78,11 @@ func main() {
 	verboseFlag := false
 	veryVerboseFlag := false
 	preventBackgroundSync := false
+	useDblockDriver := false
 	flag.BoolVar(&verboseFlag, "v", false, "Enable logging to stderr")
 	flag.BoolVar(&veryVerboseFlag, "vv", false, "Enable very verbose logging to stderr")
 	flag.BoolVar(&preventBackgroundSync, "b", false, "Disable background sync thread, for testing purposes only!")
+	flag.BoolVar(&useDblockDriver, "d", false, "Use the kmod-dblock driver rather than the default NBD driver")
```

Comment:
What are your thoughts on making this a string flag instead to avoid lots of boolean flags with won't make sense when combined (in the event of integrating N+1 usable block drivers with this utility)?

Reply:
👍 did that change, thanks!